GitUI/SpellChecker/EditNetSpell.cs

```diff
@@ -1125,5 +1125,10 @@ protected override void Dispose(bool disposing)
 
             base.Dispose(disposing);
         }
+
+        private void TextBox_MouseDown(object sender, EventArgs e)
+        {
+            TextBox.SelectionStart = TextBox.GetCharIndexFromPosition(TextBox.PointToClient(MousePosition));
+        }
```

Comment:
```suggestion
        private void TextBox_MouseDown(object sender, MouseEventArgs e)
        {
            // Set cursor only for right button here in order to avoid RichTextBox bug regarding "behind last character"
            if (e.Button == MouseButtons.Right)
            {
                TextBox.SelectionStart = TextBox.GetCharIndexFromPosition(e.Location);
            }
        }
```